src/adapters/cloudflare-worker.ts

```diff
@@ -0,0 +1,3 @@
+import { app } from "../kernel";
```

Comment:
```suggestion
export { app } from "../kernel";
```
Can't you just do that